recipes/Fisher-Callhome-Spanish/fisher_callhome_prepare.py

```diff
@@ -710,7 +710,7 @@ def remove_labels(transcription: str):
     transcription = re.sub(r"<foreign langenglishonline", "", transcription)
     transcription = re.sub(r"<foreign langenglish", "", transcription)
     transcription = re.sub(r"</foreign", "", transcription)
-    transcription = re.sub(r"<[/]?foreing\s*\w*>", "", transcription)
+    transcription = re.sub(r"<[/]?foreign\s*\w*>", "", transcription)
```

Comment:
We should make sure that this regex is still valid

Reply:
If you look at the context (before and after) all the lines are "foreign" so I'm pretty confident this one was supposed to be as well.

Reply:
Unless there's bad tags in the data that this was removing